### ISSUE TITLE
The edges of the image are foggy and blurry during the training process

### ISSUE THREAD

**Issue by LLLYLong** [noahzn/Lite-Mono#134]
Thanks to the author for such an excellent job! I encountered a problem in the process of modifying the code for training, the edges of the generated disp are blurred and the whole disp image is not displayed very clearly, I would like to consult the author what could be the reason for this。
![disp](https://github.com/noahzn/Lite-Mono/assets/100742254/6f09e8e4-1fdc-4c61-a071-b6299bcff8d9)

**Comment by noahzn** [comment]
Hi, how many rounds of epochs did you train to get this result? The result looks reasonable to me. Did you load the pre-trained ImageNet model?

**Comment by LLLYLong** [comment]
Hi, thanks for the reply from the author. 
1. This is the result after I finished the training, loaded with pre-trained weights, and looking at the predicted depth map in the senses is not very clear.
2. Here is a transcript of my training process, looking at the convergence is not particularly perfect, and I would like to ask the author if this is normal?
![loss](https://github.com/noahzn/Lite-Mono/assets/100742254/c374d228-5466-4ee8-b5bb-b39daa943338)

4. I've tried to make some modifications to change the decoder, but my training loss has become more jarring, and would like to ask some of the authors what might be wrong, and in which areas might I try to make changes?
![loss——2](https://github.com/noahzn/Lite-Mono/assets/100742254/73ca449d-199b-4958-9e63-e1af1f802d30)

**Comment by noahzn** [comment]
Hi, the figures are not accurate (see the comment here: https://github.com/noahzn/Lite-Mono/blob/main/trainer.py#L541), and they are only rough references to ensure that your training is correct. The figures look fine to me. However, if you say that you have already loaded pre-trained weights for training, then I am afraid that the depth maps should not be of such low quality. Which weights did you load? What arguments did you use for training?

**Comment by noahzn** [comment]
I'm now closing this issue as there is no response.
